lib/src/utils/matrix_localizations.dart

```diff
@@ -116,7 +116,7 @@ abstract class MatrixLocalizations {
 
   String changedTheDisplaynameTo(String targetName, String newDisplayname);
 
-  String changedTheChatPermissions(String senderName);
+  String setMeAsAdmin();
```

Comment:
can you explain more?

Reply:
it 's string used for event set me as admin, so I need to show localized string